recipes/zopfli/all/conanfile.py

```diff
@@ -8,42 +10,45 @@ class ZopfliConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/google/zopfli/"
     description = "Zopfli Compression Algorithm is a compression library programmed in C to perform very good, but slow, deflate or zlib compression."
-    topics = ("conan", )
+    topics = ("zopfli", "compression", "deflate", "gzip", "zlib")
+
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False],
-               "fPIC": [True, False]}
-    default_options = {"shared": False,
-                       "fPIC": True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
     exports_sources = ["CMakeLists.txt"]
-    generators = "cmake",
+    generators = "cmake"
     _cmake = None
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
 
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         cmake = CMake(self)
-        cmake.definitions['ZOPFLI_BUILD_INSTALL'] = True
+        cmake.definitions["ZOPFLI_BUILD_INSTALL"] = True
+        cmake.definitions["CMAKE_MACOSX_BUNDLE"] = False
```

Comment:
Is this the easiest way to fix errors due to missing `BUNDLE` from `install(...)`?

Reply:
yes